Gems/Maestro/Code/Source/Cinematics/AnimNode.h

```diff
@@ -33,6 +33,8 @@ class CAnimNode
     AZ_CLASS_ALLOCATOR(CAnimNode, AZ::SystemAllocator, 0);
     AZ_RTTI(CAnimNode, "{57736B48-5EE7-4530-8051-657ACC9BA1EE}", IAnimNode);
 
+    typedef AZStd::vector<AZStd::intrusive_ptr<IAnimTrack>> AnimTracks;
```

Comment:
Now that you're modifying this code, `using` is preferred from the guidelines: 
`using AnimTracks = AZStd::vector<AZStd::intrusive_ptr<IAnimTrack>>;`